enclave/src/main/java/org/hyperledger/besu/enclave/types/PrivacyGroup.java

```diff
@@ -82,6 +82,7 @@ public PrivacyGroup(
 
   public enum Type {
     LEGACY,
-    PANTHEON
+    PANTHEON,
+    BESU
```

Comment:
Do we need both?

Reply:
Removed.

Reply:
still here?

Reply:
Are we removing Pantheon or Besu?

Reply:
Should remove Besu. Pantheon is the only option supported by Orion atm